### PR TITLE
e2e: Identity on hover

### LINKED ISSUE #3760: E2E - Identity.vue on hover
- [ ] check some user identity on hover
- [ ] check user twitter
- [ ] check user discord
- [ ] https://github.com/dmtrKovalenko/cypress-real-events

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] E2E Test

### What's new?

- [x] PR closes #3760
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
do tests builds? 👀

**Comment by yangwao** [comment]
now do, is code good to go?

File: tests/cypress/e2e/components/identity.cy.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            shortAddress(address)
+          )
+          cy.getCy('identity-started-minting').should(
+            'contain.text',
+            startedMinting
+          )
+          cy.getCy('identity-last-bought').should('exist')
+          cy.getCy('identity-collected').should('not.contain.text', 0)
+          cy.getCy('identity-created').should('not.contain.text', 0)
+          cy.getCy('identity-sold').should('not.contain.text', 0)
```
Comment (Jarsen136) [inline tests/cypress/e2e/components/identity.cy.ts:47]: What if the sold number is 10 or 20 with the zero 0. Could it work correctly? 👀

Comment (preschian) [reply tests/cypress/e2e/components/identity.cy.ts:47]: oh, you're right, it will throw an error. it should be exact, not contains

**Review by Jarsen136 — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
some conflicts  @preschian  😇
meanwhile, tests seem to pass, great! )

**Pull request merged**